blockchain/pending.go

```diff
@@ -38,6 +38,7 @@ func (p *PendingState) ContractNonce(addr *felt.Felt) (*felt.Felt, error) {
 	if nonce, found := p.stateDiff.Nonces[*addr]; found {
 		return nonce, nil
 	} else if _, found = p.stateDiff.DeployedContracts[*addr]; found {
+		// why it's zero value for deployed contract?
```

Comment:
that is the default, when a new contract is deployed it's nonce get's initiliazed to zero.